devops/scripts/commit-type.sh

```diff
@@ -35,7 +35,7 @@ function identify_commit_type() {
 # Verifies that the given commit does not contain forbidden files.
 function only_allowed_files() {
   local commit=$1
-  local forbidden_files=("composer.lock" ".github/workflows/ci.yml")
+  local forbidden_files=("composer.lock" "package-lock.json")
```

Comment:
Why remove .github/workflows/ci.yml?

Reply:
For the immediate need, it was blocking the workflow from running on _this_ PR. However, I tried to figure out why we added it in the first place and it looked like a "¯\_(ツ)_/¯ maybe this will fix something" addition. There was no ticket associated with it and I don't really know why we would be disallowing changes to the workflow (they're non-release changes anyway).